task/packrat/src/grapefruit.rs

```diff
@@ -0,0 +1,45 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Grapefruit-specific packrat data.
+
+use task_packrat_api::HostStartupOptions;
+
+pub(crate) struct GrapefruitData {
+    host_startup_options: HostStartupOptions,
+}
+
+const fn default_host_startup_options() -> HostStartupOptions {
+    if cfg!(feature = "boot-kmdb") {
+        // We have to do this because const fn.
+        let bits = HostStartupOptions::STARTUP_KMDB.bits()
+            | HostStartupOptions::STARTUP_PROM.bits()
+            | HostStartupOptions::STARTUP_VERBOSE.bits();
```

Comment:
Can we have `HostStartupOptions::STARTUP_BOOT_RAMDISK.bits()` in this default set too please?

Reply:
Right, you did ask for that in the initial issue!  I guess we should also be building `packrat` with the `boot-kmdb` feature, to get this set of bits?

Reply:
(fixed in 8bbbda7b8e38)

Reply:
Yes, I think for grapefruit we're not likely to want to have the same concept of a non-lab image (where the bits would default to 0), at least for now.